qa/workunits/rbd/rbd_mirror_journal.sh

```diff
@@ -432,6 +432,54 @@ for i in ${image2} ${image4}; do
     remove_image_retry ${CLUSTER2} ${POOL} ${i}
 done
 
+if [ "${RBD_MIRROR_MODE}" = "snapshot" ]; then
+  testlog "TEST: request image resync when remote is not primary"
```

Comment:
In reef, we still have two separate scripts for journal-based and snapshot based mirroring -- they were merged into a single `rbd_mirror.sh` script in squid.  I'd expect this to be added to `qa/workunits/rbd/rbd_mirror_snapshot.sh` instead of `qa/workunits/rbd/rbd_mirror_journal.sh` (and without `RBD_MIRROR_MODE` check as this variable isn't even defined in reef).